fe1-web/src/features/digital-cash/reducer/DigitalCashReducer.ts

```diff
@@ -184,3 +209,37 @@ export const makeTransactionsByHashSelector = (laoId: Hash) =>
       return transactionsByHash || {};
     },
   );
+
+/**
+ * Selector for the transactions that involve given roll call tokens.
+ * @param laoId
+ * @param rollCallTokens
+ */
+export const makeTransactionsByRollCallTokenSelector = (
+  laoId: Hash,
+  rollCallTokens: RollCallToken[],
+) =>
+  createSelector(
+    (state: any) => getDigitalCashState(state).byLaoId[laoId.valueOf()]?.transactionsByHash,
+    (state: any) => getDigitalCashState(state).byLaoId[laoId.valueOf()]?.transactionsByPubHash,
+    (
+      transactionsByHash: Record<string, TransactionState> | undefined,
+      transactionsByInvPubHash: Record<string, string[]> | undefined,
```

Comment:
Same question as before :) I mean it is fine, just not sure whether it is intentional 😅